.github/workflows/rgdc.yml

```diff
@@ -0,0 +1,35 @@
+name: rgdc python client
+on:
+  push:
+    branches: "*"
+    tags: "v*"
+  pull_request:
+    branches: "**"
+jobs:
+  deploy:
+    runs-on: ubuntu-latest
+    steps:
+      - uses: actions/checkout@v2
+      - name: Set up Python
+        uses: actions/setup-python@v2
+        with:
+          python-version: "3.8"
+      - name: Install dependencies
+        run: |
+          python -m pip install --upgrade pip
+          pip install setuptools wheel twine
+      - name: Install rgdc
+        working-directory: ./rgdc
+        run: pip install -e .
+      - name: Test (not implemented)
+        working-directory: ./rgdc
+        run: python -c "from rgdc import Rgdc; print('Import went okay!')"
+      - name: Build and Publish to PyPI
+        if: github.event_name == 'push' && startsWith(github.event.ref, 'refs/tags')
```

Comment:
The if prevents the step of publishing to PyPI to only run on tags, yes.

It hasn't been tested here yet

Reply:
Or what exactly are you asking has been tested?

Reply:
I was simply asking that the if condition works correctly. I suppose we could just push a test post-release tag/version or something, I think you could do that before it's merged into master, since it targets all branches.

Reply:
It works correctly in other repos 😉 (I copy pasta'd this from where I have it working [here](https://github.com/Kitware/ipyvtklink/blob/d6d9c347a5d1451d615b27777ca9cf0f6f1070be/.github/workflows/python-publish.yml#L33))

Reply:
I suppose this isn't the best way to do this. We should really have a separate workflow only for publishing that uses the `on: push: tags: "v*"` to properly configure this condition but for now I thought having it all in one for rgdc was warranted

Reply:
I agree, but for now I think it's fine